### ISSUE TITLE
Issue with: RecoveryConsole_AllowAutomaticAdministrativeLogon

### ISSUE THREAD

**Issue by jonathan-mckinney** [MicrosoftDocs/windows-itpro-docs#2189]
The policy RecoveryConsole_AllowAutomaticAdministrativeLogon is not working correctly in Windows 10 version 1809. An error shows in Event Viewer that says: "The system cannot find the file specified."

I have other policies defined in the LocalPoliciesSecurityOptions that work fine.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 773b74b2-7dc2-d0e5-1e3b-bd6445bceda4
* Version Independent ID: 34b55057-9078-6bd0-a9ac-2f6403b84be7
* Content: [Policy CSP - LocalPoliciesSecurityOptions](https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-csp-localpoliciessecurityoptions#localpoliciessecurityoptions-recoveryconsole-allowautomaticadministrativelogon)
* Content Source: [windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md)
* Product: **w10**
* GitHub Login: @MariciaAlforque
* Microsoft Alias: **maricia**

**Comment by billgatesfan** [comment]
That is simply because MS is not updating their MS documentation even if they told us they will do so : we opened a ticket for that one in September 2018, and after few days with escalation engineers and PG, here their conclusion : 

"Symptom:
customer is currently deploying Windows 10 Enterprise RS4 in the organization, and managing with Intune
the devices are only AAD joined, and managed with Intune
customer wishes to use the Intune CSP 'Recovery console: Allow automatic administrative logon' (LocalPoliciesSecurityOptions/RecoveryConsole_AllowAutomaticAdministrativeLogon ) 
however, on any deployed device onto which this setting is deployed, the setting is not enforced
a previous case was archived from Intune Team, as the settings apparently reach the device, but are simply not honored
the issue occurs on all Windows 10 1803 devices, constantly
the issue began occurring now, as this setting was never used before
customer requests assistance so that the Intune Setting 'Recovery console: Allow automatic administrative logon' can be configured as desired

Cause:
by design.

Resolution:
functionality was retired from Product. documentation to be updated".

In short, the feature is deprecated, and this info is unfortunately also not updated here : https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/recovery-console-allow-automatic-administrative-logon

**Comment by Heidilohr** [comment]
@billgatesfan We're looking into that right now.

**Comment by BigIve** [comment]
It would be a MASSIVE help if Microsoft could update their online documentation for Intune policies.  ESPECIALLY if they have depreciated a policy, but even ones that are not, it would be nice to know the correct data formats and if they were active.
Please advise us poor Microsoft Partners when you decide to grace us with accurate and reliable documentation.

**Comment by mypil** [comment]
@officedocsbot assign @mypil

**Comment by mypil** [comment]
@jonathan-mckinney @billgatesfan @BigIve - Thank you for submitting feedback.

I will get this issue over to the Win10 ITPro writing team for investigation.

Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.

**Comment by mypil** [comment]
@jonathan-mckinney @billgatesfan @BigIve - From our understanding, the issue has been resolved but it may take a few days for the merged content to appear in the article. If you feel it hasn't been resolved, please re-open this issue.

Thank you for your contribution to make the docs better! Much appreciated!

**Comment by mypil** [comment]
@officedocsbot close
